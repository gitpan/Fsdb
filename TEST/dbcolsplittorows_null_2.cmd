prog='dbcolsplittorows'
args='-e - name'
cmp='diff -cb '
