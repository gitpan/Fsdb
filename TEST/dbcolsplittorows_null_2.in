#h name uid
# should make 3 lines
John__Heidemann  2274
# should make 3 lines
Greg_Johnson_    2275
# should make 2 lines
_Root    0
# check for duplicate entries; should make 4 lines
a___b 1
# expect 1 line (special case)
_ 1
# expect 1 lines (special case)
__
# this is a simple database
# modified to test various corner cases
#  | dbcol fullname uid
#  | dbcolrename fullname name
