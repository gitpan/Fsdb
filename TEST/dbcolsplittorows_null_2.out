#fsdb name uid
# should make 3 lines
John	2274
-	2274
Heidemann	2274
# should make 3 lines
Greg	2275
Johnson	2275
# should make 2 lines
-	0
Root	0
# check for duplicate entries; should make 4 lines
a	1
-	1
-	1
b	1
# expect 1 line (special case)
-	1
# expect 1 lines (special case)
-
# this is a simple database
# modified to test various corner cases
#  | dbcol fullname uid
#  | dbcolrename fullname name
#   | dbcolsplittorows -e - name
